### PR TITLE
[Jan 2023 rotation] New scriptworker ed25519 key

### PR DESCRIPTION
**Pull request by hneiva**

### CODE REVIEW

**Review by gabrielBusta — APPROVED** [review]

**Review by gabrielBusta — CHANGES_REQUESTED** [review]
Need to dig into those test failures

**Comment by ahal** [comment]
Odd, the commit I made for this last time was pretty much identical: https://github.com/mozilla-releng/scriptworker/commit/27f58ba128a004ebe268c7b6b9c11dffb34a0de8 (other than black formatting)

**Comment by gabrielBusta** [comment]
How odd...

**Comment by ahal** [comment]
Looks like this is a Python change:
https://github.com/python/cpython/commit/1d4d677d1c90fcf4886ded0bf04b8f9d5b60b909

Changing the method to `assert_called_once_with` should fix it.

**Comment by ahal** [comment]
I filed https://github.com/mozilla-releng/scriptworker/issues/584 to make the CI more reproducible.

File: tests/test_utils.py [hunk 1]
```diff
@@ -248,10 +248,10 @@ async def decorated_always_fail(*args, **kwargs):
 
 # create_temp_creds {{{1
 def test_create_temp_creds():
-    with mock.patch.object(utils, "createTemporaryCredentials") as p:
-        p.return_value = {"one": b"one", "two": "two"}
+    with mock.patch.object(utils, "createTemporaryCredentials") as patched:
```
Comment (hneiva) [inline tests/test_utils.py:251]: renamed `p` to `patched` as it conflicts with pdb `p` command; took me longer than I'd like to admit to realize this is why I had issues debugging it 🥲

**Review by ahal — APPROVED** [review]

**Review by gabrielBusta — APPROVED** [review]

**Pull request merged**